WordPress/Classes/ViewRelated/Themes/ThemeBrowserHeaderView.swift

```diff
@@ -28,6 +30,22 @@ open class ThemeBrowserHeaderView: UICollectionReusableView {
     fileprivate var theme: Theme? {
         didSet {
             currentThemeName.text = theme?.name
+            if let theme = theme, theme.custom {
```

Comment:
I was not happy about this code either, I think what you suggested looks much better, going to make the change, thanks :)